cpp/include/cudf/copying.hpp

```diff
@@ -26,6 +26,13 @@
 
 namespace cudf {
 namespace experimental {
+/**
+ * @brief Copy APIs (e.g. gather, scatter, copy, slice, split)
+ * @ingroup column_apis
```

Comment:
These are not just column APIs

Reply:
The `column_apis` is just a reference name and not shown anywhere.